README.md

```diff
@@ -13,170 +13,193 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
 # Overview
 
 * Time Duration is how long something lasts, from the start to end. It can be represented by a single time unit or multiple ones. 
-  - For example,
+  + For example, 
```

Comment:
Did you change the bullets on purpose? `+` is not a bullet point in markdown, I would suggest that you either revert back to `-` or use `*`.

Reply:
I used a markdown plugin from VSCode and you can clearly see the bullets in the preview mood.

Reply:
I misunderstood.